pkg/controller/baremetalhost/baremetalhost_controller.go

```diff
@@ -133,6 +133,64 @@ func (info *reconcileInfo) publishEvent(reason, message string) {
 	info.events = append(info.events, info.host.NewEvent(reason, message))
 }
 
+// hostConfigData is an implementation of host configuration data interface.
```

Comment:
It wouldn't hurt to put this in a separate file. This one is huge already.

Reply:
Done